dist/src/main/java/io/camunda/zeebe/shared/MainSupport.java

```diff
@@ -32,12 +32,6 @@ public static void setDefaultGlobalConfiguration() {
     Thread.setDefaultUncaughtExceptionHandler(
         FatalErrorHandler.uncaughtExceptionHandler(Loggers.SYSTEM_LOGGER));
 
-    // some of the health indicators use scheduled tasks which rely on the reactor schedulers, which
-    // by default have a very high bound. we only use the embedded server for management purposes,
-    // not for actual business logic, so we don't need so many
-    putSystemPropertyIfAbsent(
-        "reactor.schedulers.defaultBoundedElasticSize",
-        String.valueOf(2 * Runtime.getRuntime().availableProcessors()));
```

Comment:
:thought_balloon: We were explicitly reducing the number of threads because the management server didn't need so many. But in this case, I don't know how many we need for the REST API, so let's just leave it to the defaults.